guides/common/docinfo.html

```diff
@@ -1,8 +1,33 @@
 <script>
 const rl = document.location.toString();
+ifdef::ribbon[]
 if (rl.includes("/master/") || rl.includes("/nightly/") || rl.includes("file://")) {
   document.open();
   document.write("<span id=\"ribbon\"><a href=\"https://github.com/theforeman/foreman-documentation/tree/master/guides\">Nightly</a></span>");
   document.close();
+} else if (rl.includes("/3.1/") {
+  document.open();
+  document.write("<span id=\"ribbon\"><a href=\"https://github.com/theforeman/foreman-documentation/tree/master/guides\">RC</a></span>");
+  document.close();
+} else if (
+  rl.includes("/Installing_Proxy_on_Red_Hat/index-foreman-el.html" ||
+  rl.includes("/Installing_Proxy_on_Debian/index-foreman-deb.html" ||
+  rl.includes("/Planning_Guide/index-foreman-el.html" ||
+  rl.includes("/Planning_Guide/index-foreman-deb.html" ||
+  rl.includes("/Upgrading_and_Updating/index-foreman-el.html" ||
+  rl.includes("/Upgrading_and_Updating/index-foreman-deb.html" ||
+  rl.includes("/Configuring_Load_Balancer/index-foreman-el.html" ||
+  rl.includes("/Configuring_Load_Balancer/index-foreman-deb.html" {
```

Comment:
This could be a regexp, but I specifically want this to be just a list so we can easily be deleting guides which are considered finished.

Reply:
I guess this is fine for now IMHO.